src/modules/components/proposal/proposalVoting/proposalVotingBreakdownToken/proposalVotingBreakdownToken.test.tsx

```diff
@@ -63,30 +63,34 @@ describe('<ProposalVotingBreakdownToken /> component', () => {
         });
     });
 
-    it('correctly renders the current winning option with support indicator', () => {
+    it('correctly renders the support percentage and indicator', () => {
         const totalYes = 2400;
         const totalNo = 5000;
         const totalAbstain = 2600;
         const supportThreshold = 15;
         const tokenSymbol = 'TTT';
         render(createTestComponent({ totalAbstain, totalNo, totalYes, supportThreshold, tokenSymbol }));
 
+        const countableVotes = totalYes + totalNo;
+        const supportPercentage = countableVotes > 0 ? (totalYes / countableVotes) * 100 : 0;
+        const roundedSupportPercentage = Math.round(supportPercentage).toString();
```

Comment:
No need to check for `countableVotes > 0`
```suggestion
        const roundedSupportPercentage = Math.round((totalYes / countableVotes) * 100).toString();
```

Reply:
I believe we still need to have raw supportPercentage as a const to check the progress bar value which in this case is something like 32.434343434343 but then our formatter gives us the rounded pattern.

Reply:
Makes sense 👍🏻